File: client/Identity.WebAPI/src/app/modules/account/model/ValidateTwoFactorAuthRequest.ts
import { TwoFactorAuthModel } from "./TwoFactorAuthModel";

export class ValidateTwoFactorAuthRequest {
    email: string;
    code: string;
    method: string; // "Email" ou "Phone"
  
    constructor(code:string, model:TwoFactorAuthModel) {
      this.email = model.email;
      this.code = code;
      this.method = model.method;
    }
  }
  

File: server/Identity.domain/Domain/ValidateTwoFactorAuthRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Identity.Domain.Domain
{
    public class ValidateTwoFactorAuthRequest
    {
        public string Email { get; set; }
        public string Code { get; set; }
        public string Method { get; set; } // "Email" ou "Phone"
    }
}
